src/SFA.DAS.EmployerIncentives.Domain.UnitTests/ApprenticeshipIncentive/WhenCalculateEarningsInPhase2.cs

```diff
@@ -9,11 +9,11 @@
 using SFA.DAS.EmployerIncentives.Domain.Interfaces;
 using SFA.DAS.EmployerIncentives.Domain.ValueObjects;
 using SFA.DAS.EmployerIncentives.Enums;
-using SFA.DAS.EmployerIncentives.UnitTests.Shared.Builders;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.UnitTests.Shared.Builders;
```

Comment:
is this actually needed?

Reply:
It's the namespace where the unit test utility class IncentivePaymentProfileListBuilder resides. There was a separate utility class under the SFA.DAS.EmployerIncentives.UnitTests.Shared.Builders.Configuration namespace, but that has now been removed.